### PR TITLE
Draft: clarify noobaa usage and terms (WIP)

### PR DESCRIPTION
**Pull request by dmesser**
- use noobaa terminology and support options
- clarify noobaa limitations and alternatives
- provide details around Noobaa backing store

### CODE REVIEW

**Comment by gabriel-rh** [comment]
https://gabriel-rh.github.io/quay-docs/downstream/deploy_quay_on_openshift_op_tng/html-single/#deploy-quay-openshift-operator-tng

For review purposes...

File: modules/proc_deploy-quay-openshift-operator-tng-install.adoc [hunk 1]
```diff
@@ -43,20 +71,67 @@ spec:
...
+  type: pv-pool
+```
+---
+<1> The overall capacity of the object storage service, adjust as needed
+<2> The `StorageClass` to use for the `PersistentVolumes` requested, delete this property to use the cluster default
++
+. Apply the configuration with the following command:
++
+```
+$ oc create -f noobaa-pv-backing-store.yaml.yaml
```
Comment (dongboyan77) [inline modules/proc_deploy-quay-openshift-operator-tng-install.adoc:120]: typo `noobaa-pv-backing-store.yaml`

**Comment by gabriel-rh** [comment]
Preview here:

https://gabriel-rh.github.io/quay-docs/downstream/deploy_quay_on_openshift_op_tng/html-single/#before_installing_the_quay_operator

**Pull request merged**